🤖 Auto-merge documentation from branches

buger opened the pull request:
### **User description**
## 📚 Documentation Merge Summary

This PR contains automatically merged documentation from multiple branches.

**Triggered by:**
- **Branch:** N/A
- **Commit:** N/A
- **PR:** N/A (direct push)

**Deployment Details:**
- **Generated from:** `branches-config.json`  
- **Run ID:** 20053531825
- **Subfolder:** `(root deployment)`
- **Timestamp:** $(date)

### Changes Include:
- ✅ Merged documentation from multiple branches
- ✅ Generated unified `docs.json` configuration
- ✅ Updated assets and content structure
- ✅ Cleaned up outdated version folders

---

🚦 **This PR will be processed by merge queue to ensure proper validation and ordering.**

Previous deployment PRs have been automatically closed to prevent conflicts.


___

### **PR Type**
Documentation


___

### **Description**
- Clarify `max_idle_connections_per_host` behavior

- Add Go default and production recommendation

- Update docs across 5.8, 5.10, nightly


___

### Diagram Walkthrough


```mermaid
flowchart LR
  cfg["Gateway config docs"] -- "update parameter description" --> maxidle["max_idle_connections_per_host"]
  maxidle -- "document Go default (2)" --> godef["Go stdlib default"]
  maxidle -- "recommend 500 for production" --> rec["Production guidance"]
  cfg -- "versions updated" --> vers["5.8 / 5.10 / nightly"]
```



<details> <summary><h3> File Walkthrough</h3></summary>

<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>gateway-config.mdx</strong><dd><code>Update 5.10 docs for idle connections per host</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

snippets/5.10/gateway-config.mdx

<ul><li>Expand <code>max_idle_connections_per_host</code> description.<br> <li> Note <code>0</code> uses Go default (2).<br> <li> Recommend <code>500</code> for production.</ul>


</details>


  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/1166/files#diff-70282ae533c4e9cac1780d2bc879cd3fba4da658b5c3703ad4ee04254a8c2e8c">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>gateway-config.mdx</strong><dd><code>Clarify 5.8 idle connections per host</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

snippets/5.8/gateway-config.mdx

<ul><li>Clarify <code>max_idle_connections_per_host</code> behavior.<br> <li> Add Go default when set to <code>0</code>.<br> <li> Provide production recommendation of <code>500</code>.</ul>


</details>


  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/1166/files#diff-6a2c05175f25667d7b5588b48367248f0d5fc5ea49d3c87f7adc045c587123b7">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>gateway-config.mdx</strong><dd><code>Nightly docs: idle connections guidance</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

snippets/nightly/gateway-config.mdx

<ul><li>Improve <code>max_idle_connections_per_host</code> explanation.<br> <li> Document <code>0</code> -> Go default (2).<br> <li> Recommend <code>500</code> for production setups.</ul>


</details>


  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/1166/files#diff-596c5fdf1e0630d301e65130addcc2840e286e4476711ad922eee8e54e0673cf">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>
</table></td></tr></tr></tbody></table>

</details>

___

buger commented on tyk-docs/content/getting-started/key-concepts/rbac.md:
```diff
@@ -0,0 +1,78 @@
+
+API Ownership is enabled for all Tyk SaaS users.
+
+## API Ownership setup example
```
Should be `###` nested level

marksou replied:
Updated now

buger commented on tyk-docs/content/getting-started/key-concepts/rbac.md:
```diff
@@ -0,0 +1,78 @@
+
+In order to enable API Ownership for On-Premises installations, you need to set `enable_ownership` to `true` in your `tyk_analytics.conf` or set the `TYK_DB_ENABLEOWNERSHIP` environment variable. 
+
+API Ownership is enabled for all Tyk SaaS users.
```
Can we add this video https://www.youtube.com/watch?v=DiNxRZwtx18? 
Not the best one, but still shows concept.

marksou replied:
Ahmet was reluctant to include that. I have added it to the spreadsheet for it to be re-done by Paul and Sredny.

buger replied:
Sure, that's fine.

buger commented:
Looks good!

buger committed:
🤖 Auto-merge documentation from branches

Generated from branches-config.json at

- Merged documentation from multiple branches
- Generated unified docs.json
- Updated assets and content structure
